For Advanced/060.md

```diff
@@ -2,7 +2,7 @@
 
 Providing any credentials in plain text in your code is not reasonable, to say at least. It would be great to mask this data somehow — or better store it in a safe place and retrieve it when necessary without revealing it.
 
-G1ANT.Robot provides such a safe storage for all sensitive information: [Credential Container](g1ant.robot-window/auxiliary-windows/credential-container.md) is a tool for entering and storing data that shouldn’t be visible to everyone, who has an access to a robot script.
+G1ANT.Robot provides such a safe storage for all sensitive information: [Credential Container](https://manual.g1ant.com/link/G1ANT.Tutorials/For%20Advanced/g1ant.robot-window/auxiliary-windows/credential-container.md) is a tool for entering and storing data that shouldn’t be visible to everyone, who has an access to a robot script.
```

Comment:
The link does not work.

Reply:
wrong logic for this link - Manual repo, not Tutorial